### PR TITLE
rephrase readme

### PR DESCRIPTION
**Pull request by jscanvic**
The current README contains bullet points of features introduced by the phrase "with deepinv you can". This reads awkwardly as one might expect the verb can to be followed by a verb phrase instead of a noun phrase. I suggest using another phrase to introduce the bullet points, making the README more pleasant to read.

How it currently reads

* With deepinv you can large collection of predefined imaging operators (MRI, CT, deblurring, inpainting, etc.)
* With deepinv you can training losses for inverse problems (self-supervised learning, regularization, etc.)
* With deepinv you can many pretrained deep denoisers which can be used for plug-and-play restoration

How it might read instead

* Using deepinv gives you access to a large collection of predefined imaging operators (MRI, CT, deblurring, inpainting, etc.)
* Using deepinv gives you access to training losses for inverse problems (self-supervised learning, regularization, etc.)
* Using deepinv gives you access to many pretrained deep denoisers which can be used for plug-and-play restoration

### Checks to be done before submitting your PR
- [ ] `python3 -m pytest tests/` runs successfully.
- [ ] `black .` runs successfully.
- [ ] `make html` runs successfully (in the `docs/` directory).
- [ ] Updated docstrings related to the changes (as applicable).
- [x] Added an entry to the [CHANGELOG.rst](../CHANGELOG.rst).

### CODE REVIEW

**Review by matthieutrs — COMMENTED** [review]
Thanks for raising this! LGTM, @tachella what do you think?

File: README.rst [hunk 1]
```diff
@@ -11,16 +11,16 @@ Introduction
 ------------
 Deep Inverse is an open-source pytorch library for solving imaging inverse problems using deep learning. The goal of ``deepinv`` is to accelerate the development of deep learning based methods for imaging inverse problems, by combining popular learning-based reconstruction approaches in a common and simplified framework, standardizing forward imaging models and simplifying the creation of imaging datasets.
 
-With ``deepinv`` you can:
+Using ``deepinv`` gives you access to
```
Comment (tachella) [inline README.rst:14]: I would go for something shorter like:

- ``deepinv`` features

**Review by tachella — CHANGES_REQUESTED** [review]
I'm happy with the changes but I would simplify the first line (see comment)

**Comment by jscanvic** [comment]
Thanks both for reviewing the PR! It should be good now.

**Pull request merged**